File: server/router_auth.js
const Login = require('./con/login');
const Register = require('./con/register');
const RegisterDetail = require('./con/register_detail');
const routerAuth = require('express').Router();

routerAuth.post('/register', Register.sekarang);
routerAuth.post('/login', Login.sekarang);
routerAuth.post('/register-detail', RegisterDetail.sekarang);

routerAuth.post('/login-device', Login.device)
routerAuth.post('/login-cashier', Login.loginCashier)

module.exports = routerAuth;

File: client/lib/utils/router_auth.dart
import 'package:http/http.dart'
as http;

import 'package:propos/utils/config.dart';

class RouterAuth {

    static Future < http.Response > register(Map body) => http.post(Uri.parse("${Config.host}/auth/register"), body: body);

    static Future < http.Response > login(Map body) => http.post(Uri.parse("${Config.host}/auth/login"), body: body);

    static Future < http.Response > registerDetail(Map body) => http.post(Uri.parse("${Config.host}/auth/register-detail"), body: body);

    static Future < http.Response > loginDevice(Map body) => http.post(Uri.parse("${Config.host}/auth/login-device"), body: body);

    static Future < http.Response > loginCashier(Map body) => http.post(Uri.parse("${Config.host}/auth/login-cashier"), body: body);

}